src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -9754,8 +9754,14 @@ namespace AirflowNetworkBalanceManager {
                 for (j = 1; j <= NumOfZones; ++j) {
                     if (!ZoneEquipConfig(j).IsControlled) continue;
                     if (ZoneEquipConfig(j).ZoneNode == i) {
-                        NodeFound(i) = true;
-                        AirflowNetworkNodeData(ZoneEquipConfig(j).ActualZoneNum).EPlusNodeNum = i;
+                        if (ZoneEquipConfig(j).ActualZoneNum > AirflowNetworkNumOfNodes) {
+                            ShowSevereError(RoutineName + NodeID(i) + " is not defined as an AirflowNetwork:Distribution:Node object.");
```

Comment:
If more work is to be done on this PR I'd suggest quoting the `NodeID(i)` since it could be named in a variety of ways and may look funny in part of the sentence.  If no more work is needed I won't require that change.